src/css/map-view.css

```diff
@@ -1289,76 +1281,64 @@ other class: .ui-slider-range */
 }
 
 .nav-help {
-  /* Temporary. Background can't be buttons-bkg because the hand image on the button is white with no outline. */
-  background-color: var(--portal-grey-4, var(--map-col-bkg-lighter__deprecate));
-  color: white;
-
-  border-radius: var(--map-border-radius-big);
+  color: var(--map-col-text-label, white);
 }
 
 .nav-help .map-view__button {
-  /* Temporary. Background can't be buttons-bkg because the hand image on the button is white with no outline. */
-  background-color: var(--portal-grey-4, var(--map-col-bkg-lighter__deprecate));
+  background-color: var(--map-col-content-bkg-highlight, var(--map-col-bkg-lighter__deprecate));
+  border-radius: var(--map-border-radius-small);
   color: var(--map-col-text-label);
 
   padding: 0.65rem 0.9rem;
   border-bottom-left-radius: 0;
   border-bottom-right-radius: 0;
-  opacity: var(--map-no-brightness-or-opacity-tweaks, 0.8);
 }
 
 .nav-help .map-view__button--active {
-  /* Temporary. Background can't be buttons-bkg because the hand image on the button is white with no outline. */
-  background-color: var(--portal-grey-5, var(--map-col-bkg-lightest__deprecate));
+  /*
+  Temporary. Background needs to be a darker color because the hand image on the button
+  is white with no outline. Should be updated to --map-col-buttons-bkg-highlight when
+  implementing the design at
+  https://www.figma.com/file/9R80DBDAMQtOsu8y1HkAVT/permafrost-discovery-gateway?type=design&node-id=257-20734&mode=design&t=OloKsPrtR72yOcqn-4
```

Comment:
This is likely not accessible to all users of this repo, should we add a screenshot to the repo instead of linking to Figma?

Reply:
Same with below link to Figma, alternatively, link to a GitHub issue as the TODO instead of figma

Reply:
Converted to GH issue and TODO